### ISSUE TITLE
use  nsexec do export is not valid

### ISSUE THREAD

**Issue by zhqqqy** [chaos-mesh/nsexec#1]
When I execute the below command
```shell
nsexec  -m /proc/46983/ns/mnt -p /proc/46983/ns/pid sh -c  "export"
```
It shows not the environment variables inside the container, but the local environment variables.

**Comment by YangKeao** [comment]
Yes. It is expected. Only the process forked from the first process of the container will have its environment variables. The `nsenter` has the same behavior. 

Please try `env "$(xargs -0 -L1 -a /proc/46983/environ)" nsexec  -m /proc/46983/ns/mnt -p /proc/46983/ns/pid sh -c  "export" `. I'm not sure whether it will work perfectly 🤔
